### PR TITLE
feat :added wishlist page

### PR DESCRIPTION
**Pull request by Tanjum786**
**Added wish list to the E-commerce website**
Screenshot from 2022-02-16 16-58-26

### CODE REVIEW

File: CSS/wishlist.css [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+}
+
+.price_tag,
+.e_secondary_btn,
+.wishlist_card_details {
+    padding: 0.5rem;
+}
+
+.wishlist_container {
+    background-color: #f9fafb;
```
Comment (nishasen) [inline CSS/wishlist.css]: use variable here

File: CSS/wishlist.css [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+    font-size: 2rem;
+    font-family: var(--secondary-font);
+    margin-top: 1rem;
+}
+
+.wishlist_card {
+    width: 20rem;
+    box-shadow: 1px 1px 4px 1px var(--secondary-color);
+    border-radius: 0.5rem;
+    background-color: white;
```
Comment (nishasen) [inline CSS/wishlist.css]: use variable here

File: CSS/wishlist.css [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+}
+
+.price_container {
+    flex-direction: column;
+}
+
+.e_secondary_btn {
+    width: 100%;
+    border-radius: 0.4rem;
+    color: #60a5fa;
```
Comment (nishasen) [inline CSS/wishlist.css]: use variable

File: CSS/wishlist.css [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+    background-color: var(--white-color);
+    border: 1px solid #60a5fa;
+    background-color: var(--white-color);
+}
+
+.e_secondary_btn:hover {
+    background-color: #60a5fa;
+    color: white;
```
Comment (Sameony) [inline CSS/wishlist.css]: Do properly align the file before merging. You can use the prettier or beautify extension for that

File: CSS/wishlist.css [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+    width: 100%;
+    border-radius: 0.4rem;
+    color: #60a5fa;
+    background-color: var(--white-color);
+    border: 1px solid #60a5fa;
+    background-color: var(--white-color);
+}
+
+.e_secondary_btn:hover {
+    background-color: #60a5fa;
```
Comment (Sameony) [inline CSS/wishlist.css]: create and use color variables for better readability and consistency

**Review by Sameony — COMMENTED** [review]
Suggested some minor refactoring changes

**Review by nishasen — COMMENTED** [review]
Hey everything looks good, at some places you have used solid color names, instead use variables there.

File: CSS/wishlist.css (hunk 4 above)
Comment (Tanjum786) [reply CSS/wishlist.css]: okay will do that

File: CSS/wishlist.css (hunk 5 above)
Comment (Tanjum786) [reply CSS/wishlist.css]: yes, I will create

File: CSS/wishlist.css (hunk 3 above)
Comment (Tanjum786) [reply CSS/wishlist.css]: yeah, I will work on it.

**Review by nishasen — APPROVED** [review]

**Review by Sameony — APPROVED** [review]

**Pull request merged**